config/authors/mesmacosta.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Marcelo Costa",
+  "bio": "software engineer & google cloud certified architect and data engineer | love to code, working with open source and writing @ alvin.ai  ",
```

Comment:
Software Engineer & Google Cloud Certified Architect And Data Engineer | Working With Open Source And Writing @ Alvin.ai